src/index.ts

```diff
@@ -25,16 +25,14 @@ export async function run() {
     core.setOutput("number", newPr.number);
     core.setOutput("url", newPr.html_url);
 
-    if (labels.length > 0) {
+    if (Array.isArray(labels) && labels.length > 0) {
```

Comment:
```suggestion
    if (labels.length > 0) {
```
If we return an empty array this won't be needed.